data.py

```diff
@@ -82,7 +82,7 @@ def populate(oxigraph_url=None, write_to_db=True):
     # TODO - need to iterate all files in ./edtions not just the one
 
     # Load from disk
-    editions_source_path = Path(subbed_metadata_store_content_path / "editions/cpih_2022-01.json")
+    editions_source_path = Path(subbed_metadata_store_content_path / "editions/beis_2005-2019_la_carbon_emission.json")
```

Comment:
dont change this please, this codes been updated, it'll scoop up everything in the /editions directory. You wont need to finagle the python.